packages/material-ui/src/locale/index.js

```diff
@@ -28,6 +28,35 @@ export const azAZ = {
   },
 };
 
+export const csCZ = {
+  props: {
+    MuiTablePagination: {
+      backIconButtonText: 'Předchozí stránka',
+      labelRowsPerPage: 'Řádků na stránce:',
+      labelDisplayedRows: ({ from, to, count }) => `${from}-${to === -1 ? count : to} z ${count}`,
+      nextIconButtonText: 'Další stránka',
+    },
+    MuiRating: {
+      getLabelText: value => {
+        if (value === 1) {
+          return `${value} Hvězda`;
```

Comment:
I think we use "hvězdička" more in this context than "hvězda":

- in comments under films at csfd.cz (e.g. [this](https://www.csfd.cz/film/8283-posledni-skaut/komentare/)) where rating is done by stars you can find people using both,
- but I think in adjectives it's never used with "hvězda" in this context, always in the diminutive form: e.g. "pětihvězdičkový hotel" would never be "pětihvězdový hotel" or at least it sounds somewhat weird

So I'd vote for using "1 hvězdička / 2-3 hvězdičky / 4+ hvězdiček" even though it sounds childish in a way. I think in the context of rating it's more correct / more used.

Reply:
Sounds about right. I'll make the change. Thank you!

Reply:
I've also aligned SK to reflect the change in CZ